cloudbuild.yaml

```diff
@@ -1,20 +1,12 @@
 timeout: 3600s
 steps:
-- id: certbot
-  name: 'gcr.io/cloud-builders/docker'
-  args: ["build", "-t", "us.gcr.io/$PROJECT_ID/certbot:latest", "."]
-  dir: .
-  waitFor: ['-']
-
 - id: certbot-dns
```

Comment:
this build will now fail since it relies on `certbot` I think

Reply:
In that case, should I also delete certbot-dns-google/Dockerfile and the corresponding code in cloudbuild.yaml? Should I completely delete cloud build.yaml in that case, or still leave the very first and last line?